src/main/java/edu/kathieRoy/advancedjava/StockQuoteFactory.java

```diff
@@ -0,0 +1,27 @@
+package edu.kathieRoy.advancedjava;
+
+import java.math.BigDecimal;
+import java.util.Calendar;
+
+/**
+ * @author kathie Use the class to get the object of StockQuote
```

Comment:
this description is a bit off. 
this factory gets the stock service implementation which can used to get stockquote info.